src/components/Contact/Contact.js

```diff
@@ -4,15 +4,16 @@ import { StyleSheet, Text, View, ImageBackground } from 'react-native';
 
 // Components
 import Header from './Header';
+import Chat from './Chat';
+import Help from './Help';
 
 export default class Home extends React.Component {
   render() {
     return (
         <ImageBackground source={require('../../img/img-bg-dark.jpg')} style={styles.imageBackground}>
           <Header />
-
-
-
+          <Chat />
+          <Help />
```

Comment:
Contact/Contact.js: Code is clean and well-organized for future additions/amendments.